docs/antrea-proxy.md

```diff
@@ -253,6 +253,24 @@ data:
       proxyLoadBalancerIPs: false
 ```
 
+It's worth noting that AntreaProxy will bypass all the external `loadBalancerIP`s
+with the above configuration. After K8s v1.29, field `LoadBalancerIPMode` was
+introduced to `LoadBalancerIngress` in Service status, providing users with a
+more-fined mechanism to control how a corresponding external `loadBalancerIP` behaves.
+
+- If the value of `LoadBalancerIPMode` is `LoadBalancerIPModeVIP` or nil, the
+  traffic destined for the external `loadBalancerIP` should behave the same as
+  current implementation, getting load-balanced at the source Node.
```

Comment:
s/should behave the same as current implementation, getting load-balanced at the source Node/should follow the default behavior and get load-balanced at the source Node.